pmoves/services/spark-shape-worker/README.md

```
@@ -0,0 +1,39 @@
+export SPARK_SHAPE_SECRET="$(openssl rand -hex 32)"
+
+export NATS_URL="nats://nats:pmoves@nats:4222"
+python3 main.py
+```
+
+## Configuration
+
+| Env var | Default | Purpose |
+|---|---|---|
+| `NATS_URL` | `nats://nats:pmoves@nats:4222` | NATS connection URL |
+| `SPARK_SHAPE_SECRET` | `
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)